app/backend/bin/cli.py

```diff
@@ -0,0 +1,556 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'lib', 'boto'))
+
+import logging
+import traceback
+import json
+import re
+import pprint
+import time
+import uuid
+import datetime
+import tempfile
+import subprocess
+import shlex
+import argparse
+import boto
+import uuid
+import threading
+
+from databases.dynamo_db import DynamoDB
+import tasks
+from tasks import *
+
+from common.config import CeleryConfig, JobTypes, JobDatabaseConfig, JobConfig, AgentTypes, AWSConfig
+import common.helper as helper
+
+import celery
+from celery.task.control import inspect
+
+
+def get_aws_credentials():
```

Comment:
The environment variables I have are named: AWS_ACCESS_KEY and AWS_SECRET_KEY

It might be more compatible to check both.